frontend/actions/quiz.ts

```
@@ -1,5 +1,6 @@
 'use server';
 
+import { and, eq, inArray } from 'drizzle-orm';
 import { eq, inArray } from 'drizzle-orm';
```

Biome (2.3.13)

[error] Shouldn't redeclare 'eq'. Consider to delete it or rename it.

'eq' is defined here:


(lint/suspicious/noRedeclare)